feat: draw measurements in e.g. `.obj`, refactor and update unittests

AJPfleger opened the pull request:
A new function+unittest for visualising measurements as ellipses.

In the future, we maybe also want to add the possibility to display 1D measurements.

Refined all `EventDataView3D`-Unittests.

Fixed also some typos missed by the spellcheck.

asalzburger reviewed (changes requested):
Good for a first implementation, the 1D case would also be great - but more importantly, can you add a UnitTest in `Tests/Visualization` ?

AJPfleger commented:
> Good for a first implementation, the 1D case would also be great - but more importantly, can you add a UnitTest in `Tests/Visualization` ?

I added 2 unit tests checking for errors in the output and also comparing the length (lines) of the output to a reference value. I also added the length-check to the 4 other tests of the same kind.

asalzburger reviewed (approved):
And off we go.
